test/EndToEndTests/Common/Microsoft.OData.Client.E2E.TestCommon/EndToEndTestBaseOfT.cs

```diff
@@ -0,0 +1,58 @@
+﻿//-----------------------------------------------------------------------------
+// <copyright file="EndToEndTestBaseOfT.cs" company=".NET Foundation">
+//      Copyright (c) .NET Foundation and Contributors. All rights reserved.
+//      See License.txt in the project root for license information.
+// </copyright>
+//------------------------------------------------------------------------------
+
+using Microsoft.Extensions.DependencyInjection;
+using Xunit;
+
+namespace Microsoft.OData.Client.E2E.TestCommon
+{
+    public class EndToEndTestBase<T> : IClassFixture<TestWebApplicationFactory<T>> where T : class
+    {
+        private HttpClient _client;
+        private readonly IHttpClientFactory _httpClientFactory;
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="WebODataTestBase{TStartup}"/> class.

```

Comment:
What is `WebODataTestBase{TStartup}`? Is that reference a relic from the past that should be removed?

Reply:
This was a type. Fixed.